charts/aac-manage-case-assignment/values.preview.template.yaml

```diff
@@ -14,7 +14,7 @@ java:
         - notify-mca-api-key-secret
   environment:
     S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
-    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-1260-java.ccd
-    CCD_DEFINITION_STORE_API_BASE_URL: http://ccd-definition-store-api-pr-575-java.ccd
+    CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-pr-2163-java.ccd
```

Comment:
This needs to be reverted before merging.

Reply:
Solved in [cdd8c09](https://github.com/hmcts/aac-manage-case-assignment/pull/455/commits/cdd8c09005cadb5af36a9cf1186baae088ffd5fa)